### PR TITLE
Minor. Fix whitespace in connection.cs

### PR DESCRIPTION
**Pull request by chsienki**
Carrying on from #29711, addressing @agocke feedback.

### CODE REVIEW

**Comment by chsienki** [comment]
@dotnet/roslyn-compiler Hopefully the easiest review ever, please.

**Comment by jcouv** [comment]
>                 else if(!string.Equals(request.CompilerHash, BuildProtocolConstants.GetCommitHash(), StringComparison.OrdinalIgnoreCase))

might as well fix this one too.
nit: If I were you, I would have just fixed this in whatever next PR you're lining up. Probably doesn't need a dedicated PR

---
Refers to: src/Compilers/Server/VBCSCompiler/Connection.cs:112 in a351cbf. [](commit_id = a351cbfe53c1edbdd0aaa2c27a2e96d5c0b2cce1, deletion_comment = False)

File: src/Compilers/Server/VBCSCompiler/Connection.cs [hunk 1]
```diff
@@ -105,7 +105,7 @@ public async Task<ConnectionData> HandleConnection(bool allowCompilationRequests
                     return new ConnectionData(CompletionReason.CompilationNotStarted);
                 }
                 
-                if(request.ProtocolVersion != BuildProtocolConstants.ProtocolVersion)
+                if (request.ProtocolVersion != BuildProtocolConstants.ProtocolVersion)
                 {
                     return await HandleMismatchedVersionRequest(cancellationToken).ConfigureAwait(false);
                 }
```
Comment (agocke) [inline src/Compilers/Server/VBCSCompiler/Connection.cs]: Also the line below.

Comment (jcouv) [reply src/Compilers/Server/VBCSCompiler/Connection.cs]: I went ahead and fixed that (directly through github UI)

**Comment by jcouv** [comment]
test windows_debug_spanish_unit32_prtest please

**Comment by jcouv** [comment]
test windows_debug_unit32_prtest please

**Pull request merged**